client/galaxy/style/scss/base.scss

```diff
@@ -608,7 +608,7 @@ div.toolFormRow {
         @extend .py-1;
         @extend .px-2;
         cursor: move;
-        background: $panel-bg-color;
+        background: $base-color-3;
```

Comment:
Mmh, this doesn't exist any more: https://travis-ci.org/galaxyproject/galaxy/jobs/492133195

Reply:
It was removed in https://github.com/galaxyproject/galaxy/pull/7203/files#diff-083a815db1387e189109dae6cce99198L7

Reply:
Argh, only checked against my PR, and looked fine against that, didn't realise it was later removed. Sorry, should have rebuilt the client.